micro-rdk/src/common/webrtc/ice.rs

```diff
@@ -165,10 +176,21 @@ impl ICEAgent {
         let bytes = Bytes::from(encoder.encode_into_bytes(message).unwrap());
 
         // TODO(RSDK-3063) Twilio address is hard-coded, we should support additional server via WebRTCOptions
-        let mut stun_ip = "global.stun.twilio.com:3478".to_socket_addrs().unwrap();
+        let mut stun_ip = match "global.stun.twilio.com:3478".to_socket_addrs() {
+            Ok(stun_ip) => stun_ip,
+            Err(err) => {
+                log::warn!("Failed trying to resolve STUN server address; no reflexive candidate will be generated: {}", err);
+                return Ok(());
```

Comment:
is there ever a time where we actually want to return an Error rather than Ok? Like what if we are not in an isolated network and we still can't resolve the address?

Reply:
Yes, I thought about that a bit, and also the other case, where we could add the intrinsic local candidate, and then even though we were able to resolve the STUN server, something goes wrong afterwards. In that case, we actually do return an error, but we sort of don't want to fail the method because we still did create one local candidate.

Overall, my view is that `local_candidates` probably shouldn't really be returning a `Result` at all. It is a best effort "how can I describe myself". If the answer is "I can't", it should be up to the higher level (i.e. the caller of `local_candidates` up in `api.rs`) to decide how to handle that.